numpy/random/_generator.pyx

```diff
@@ -4136,6 +4137,11 @@ cdef class Generator:
             raise ValueError('alpha <= 0')
         alpha_data = <double*>np.PyArray_DATA(alpha_arr)
 
+        alpha_csum = np.ascontiguousarray(np.cumsum(alpha[::-1])[::-1])
+        alpha_csum_arr = <np.ndarray>np.PyArray_FROM_OTF(
+            alpha_csum, np.NPY_DOUBLE, np.NPY_ALIGNED | np.NPY_ARRAY_C_CONTIGUOUS)
```

Comment:
PEP 8 also applies to Cython files, so this line needs to be split to keep the length under 80.

Reply:
Thanks for spotting. This is fixed now.